src/Pimple/KafkaSchemaRegistryApiClientProvider.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient\Pimple;
+
+use Buzz\Client\Curl;
+use Jobcloud\KafkaSchemaRegistryClient\ErrorHandler;
+use Jobcloud\KafkaSchemaRegistryClient\HttpClient;
+use Jobcloud\KafkaSchemaRegistryClient\KafkaSchemaRegistryApiApiClient;
+use LogicException;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+
+class KafkaSchemaRegistryApiClientProvider implements ServiceProviderInterface
+{
+
+    public const ROOT = 'kafka_schema_registry';
+    public const USERNAME = 'username';
```

Comment:
```suggestion
    public const SETTING_KEY_USERNAME = 'username';
```
and for the others ofc

Reply:
Do you want all to be preffixed with `setting`?